[W-10728932] New landing page (#120)
mulesoft/docs-general: modules/ROOT/pages/index.adoc

* xref:munit::index.adoc[Build automated tests for your APIs and integrations]
--

image::deploy.png[]

--
[discrete]
=== Deploy

Deploy APIs and apps to your hosting targets, and secure them with policies.

* xref:runtime-manager::deployment-strategies.adoc[Deployment options]
* xref:api-manager::getting-started-proxy.adoc[Configure a proxy for your API]
* xref:api-manager::policies-landing-page.adoc[Create policies to secure your API]
--

image::monitor.png[]

--
[discrete]
=== Monitor

Monitor your APIs and integrations using dashboards, metrics, and visualization.

* xref:monitoring::index.adoc[Monitor Anypoint Platform performance from a single user interface]
* xref:api-functional-monitoring::index.adoc[Monitor your deployed API performance]
* xref:runtime-manager::monitoring.adoc[Monitor your deployed apps]
--

== Quick Links

[#popular-topics]
=== Popular Topics

* xref:dataweave::dw-core-functions-filter.adoc[DataWeave Function: filter]
* xref:dataweave::dataweave-cookbook-format-dates.adoc[Format Dates and Times]
* xref:dataweave::dw-operators.adoc[DataWeave Operators]
* xref:connectors::index.adoc[Connectors]

> * xref:mule-runtime::mule-error-concept[Mule Errors]

Reviewer: ```suggestion
* xref:mule-runtime::mule-error-concept.adoc[Mule Errors]
```

= [.brand]#MuleSoft# Documentation
:page-layout: home
:page-fragmentize:
:!sectids:
ifndef::env-site[:imagesdir: ../images]

[#cta]

> xref:api-led-overview.adoc[New to MuleSoft? Start here]

Reviewer 2: ```suggestion
xref:api-led-overview.adoc[New to MuleSoft? Start here.]
```

[#the-road]
== The Road to Digital Transformation

image::getting-started.png[]

--
[discrete]
=== Getting Started

Build the digital transformation your business needs.

> * xref:api-led-overview.adoc[Tutorial: Build an API from Start to Finish]

Reviewer 2: ```suggestion
* xref:api-led-overview.adoc[Tutorial: Build an API from start to finish].
```

Reviewer 2: (Changed my mind - let's use sentence case here. Sorry.)

Reviewer 3: Just noticed that the period at the end of the sentence should be outside the brackets right? (e.g. **finish].**)

Reviewer 2: Yes. Good catch.

--

> image::discover.png[]

Reviewer 2: Do we need alt text for these images? A11Y?

Author: added alt text

--
[discrete]
=== Discover

Browse Exchange to find existing API specifications, templates, examples, and other assets that you can reuse for your projects.

* https://www.anypoint.mulesoft.com/exchange/[Discover reusable assets^]
--

image::design.png[]

--
[discrete]
=== Design

Create and publish new API specifications from scratch.

> * xref:design-center::design-create-publish-api-specs.adoc[Create a reusable API specification]

Reviewer 2: Do we want to tie these tasks to products? 
Create a reusable API specification with API Designer.

Reviewer 3: My only concern with this approach is that some of these tasks are already long enough to add the product's name. Also, does it add any value for the user to mention the product's name here?

Reviewer 2: I guess it's not necessary.

> * xref:design-center::design-create-publish-api-specs.adoc[Create a reusable API specification]

Reviewer 2: ```suggestion
* xref:design-center::design-create-publish-api-specs.adoc[Create a reusable API specification].
```

> * xref:access-management::index.adoc[Plan for access]

Reviewer 2: What does this mean? Plan for access to API specs?

Reviewer 3: Since this is linking to Access Management, a more accurate task name could be: **[Manage your Anypoint Platform accounts and permissions]**

Reviewer 2: I agree. Should this be in "Create and publish new API specifications"?

Reviewer 2: If this stays in this section, should it be the first item?

Manage access to your Anypoint Platform account?

> * xref:access-management::index.adoc[Plan for access]

Reviewer 2: ```suggestion
* xref:access-management::index.adoc[Manage access to your Anypoint Platform account].
```

> * xref:general::security.adoc[Plan for security]

Reviewer 2: "Understand security features of Anypoint Platform"?

Reviewer 3: "Build security into your application network" ? 
This xref is actually linking to a file that is orphaned (and I think incomplete): https://docs.mulesoft.com/general/security

I don't know if we want to keep this link or remove it. I think the best option would be to improve that **security** article to mention every security measure users can implement in MuleSoft, but I don't know who can commit to do this since it requires some time to investigate several products and mention their security offerings.

Reviewer 2: Agreed.

> * xref:general::security.adoc[Plan for security]

Reviewer 2: ```suggestion
* xref:general::security.adoc[Build security into your application network].
```

--

image::develop-and-test.png[]

--
[discrete]
=== Develop & Test

> Build and test APIs and integrations.

Reviewer 2: ```suggestion
Build and test APIs and integration apps.
```

* xref:studio::index.adoc[Develop your API]
* xref:connectors::index.adoc[Connect your data, systems, and apps]
* xref:mule-runtime::mule-app-dev.adoc[Develop the Mule app that integrates systems, services, APIs, and devices]
* xref:dataweave::index.adoc[Access and transform data within the Mule app]
* xref:munit::index.adoc[Build automated tests for your APIs and integrations]
--

image::deploy.png[]

--
[discrete]
=== Deploy

Deploy APIs and apps to your hosting targets, and secure them with policies.

> * xref:runtime-manager::deployment-strategies.adoc[Deployment options]

Reviewer 2: ```suggestion
* xref:runtime-manager::deployment-strategies.adoc[Choose a deployment option].
```

> * xref:api-manager::getting-started-proxy.adoc[Configure a proxy for your API]

Reviewer 2: ```suggestion
* xref:api-manager::api-proxy-landing-page.adoc[Configure a proxy for your API].
```

> * xref:api-manager::policies-landing-page.adoc[Create policies to secure your API]

Reviewer 2: ```suggestion
* xref:gateway::policies-overview.adoc[Create policies to secure your API].
```

--

image::monitor.png[]

--
[discrete]
=== Monitor

Monitor your APIs and integrations using dashboards, metrics, and visualization.

> * xref:monitoring::index.adoc[Monitor Anypoint Platform performance from a single user interface]

Reviewer 2: ```suggestion
* xref:monitoring::index.adoc[View metrics for integration apps and APIs].
```

> * xref:api-functional-monitoring::index.adoc[Monitor your deployed API performance]

Reviewer 2: ```suggestion
* xref:api-functional-monitoring::index.adoc[Test the functional behavior and performance of your API].
```

> * xref:monitoring::index.adoc[Monitor Anypoint Platform performance from a single user interface]
> * xref:api-functional-monitoring::index.adoc[Monitor your deployed API performance]
> * xref:runtime-manager::monitoring.adoc[Monitor your deployed apps]

Reviewer 3: ```suggestion
* xref:monitoring::index.adoc[Monitor Anypoint Platform performance from a single user interface]
* xref:api-functional-monitoring::index.adoc[Check your deployed API performance]
* xref:runtime-manager::monitoring.adoc[Supervise your deployed apps]
```

Reviewer 3: @hannanelson do you agree with this rewording?

Reviewer 2: This implies monitoring Anypoint Platform performance, not performance of integrations/applications:
`Monitor Anypoint Platform performance from a single user interface`

Reviewer 2: View metrics for integration apps?

> * xref:runtime-manager::monitoring.adoc[Monitor your deployed apps]

Reviewer 2: ```suggestion
* xref:runtime-manager::monitoring.adoc[Monitor your deployed apps].
```

Reviewer 2: or Monitor your deployed apps in Runtime Manager.

* xref:mule-runtime::batch-processing-concept.adoc[Batch Processing]
* xref:dataweave::dataweave-functions.adoc[DataWeave Reference]
* xref:connectors::index.adoc[Connectors]
// rank #8-10:
// * xref:mule-runtime::mule-error-concept.adoc[Mule Errors]
// * xref:mule-runtime::cloudhub-architecture.adoc[CloudHub Architecture]
// * xref:dataweave::dw-core-functions-contains.adoc[DataWeave Function: contains]

[#latest-releases]
=== Latest Releases

////
removing table until we can automate, keeping hidden here for future reference
[cols=10;90]
|===
|April 11
|xref:api-functional-monitoring/api-functional-monitoring-release-notes.adoc[API Functional Monitoring 2.1]

|April 11
|xref:connector/connector-db.adoc#1-13-0[Database Connector 1.13]

|April 2
|xref:design-center/design-center-release-notes-api_specs.adoc#2-41-0[API Designer (Crowd) Release Notes 2.41]

|April 2
|xref:api-mocking-service/api-mocking-service-release-notes.adoc#2-38-0[API Mocking Service Release Notes 2.38]

|March 31
|xref:studio/anypoint-studio-7.12-with-4.4-runtime-release-notes.adoc[Anypoint Studio 7.12]
|===
////

xref:#[View all release notes,role=view-all]

> 

Reviewer 2: [#archived-docs]
=== Archived Documentation

* https://archive.docs.mulesoft.com/[Archived MuleSoft Documentation^]

Author: I added the archive link at the bottom (and the left nav):

<img width="1231" alt="Screen Shot 2022-10-12 at 12 45 24 PM" src="https://user-images.githubusercontent.com/10934908/195434359-546c043c-bb21-4251-8bad-a7266adf059e.png">
